web_services/foxx/brightid/db.js

```diff
@@ -112,7 +112,7 @@ function connect(op) {
   }
 }
 
-function userConnections(userId, direction = "outbound") {
```

Comment:
Is there a reason you removed the default value for this parameter?  I think it would break backwards compatibility.

Reply:
Yes, I think having less optional parameters here is better for readability. I realized the only place that the parameter is not passed is in the tests, so I updated those function calls so the change is not breaking them.

Reply:
> Yes, I think having less optional parameters here is better for readability. I realized the only place that the parameter is not passed is in the tests, so I updated those function calls so the change is not breaking them.

We would need to bump the API version to 7 (or soulbound v2) because it's a breaking change

Reply:
I don't think so, because direction is already a `required` parameter in the HTTP API. And since the `withVerifications` parameter is optional, all the current clients should work properly.
I also didn't change any tests, and they all pass.

Reply:
Yes, you're right.  The API requires it, so it makes more sense if this function also requires it.